lib/common.sh

```diff
@@ -683,9 +686,40 @@ verify_iptable_entry()
     fi
 }
 
+# Find CheckHost value for stunnel configuration based on storage account hostname.
+get_check_host_value()
+{
+    local hostname=$1
+    local check_host_value="*.file.core.windows.net"
+
+    declare -A certs
+    certs=(
+        ["preprod.core.windows.net$"]="*.file.preprod.core.windows.net"
+        ["chinacloudapi.cn$"]="*.file.core.chinacloudapi.cn"
+        ["usgovcloudapi.net$"]="*.file.core.usgovcloudapi.net"
+    )
+
+    for cert in "${!certs[@]}"; do
+        if [[ "$hostname" =~ $cert ]]; then
+                check_host_value="${certs[$cert]}"
+                break
+        fi
+    done
+
+    echo "$check_host_value"
+}
+
 # On some distros mount program doesn't pass correct PATH variable.
 export PATH=$PATH:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
+if command -v netstat &> /dev/null; then
+    NETSTATCOMMAND="netstat"
+elif command -v ss &> /dev/null; then
+    NETSTATCOMMAND="ss"
+else
+    eecho "[FATAL] Neither 'netstat' nor 'ss' is available!"
+    exit 1
+fi
```

Comment:
since v3 is not using it, let's not make this a fatal error. v4 can check for it and bail out if it's not set.